.github/workflows/release.yml

```diff
@@ -0,0 +1,135 @@
+
+name: Release
+# This workflow builds the wheels "on tag".
+# If run from the hyperspy/hyperspy repository, the wheels will be uploaded to pypi ;
+# otherwise, the wheels will be available as a github artefact.
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+    - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+
+jobs:
+  create_release:
+    name: Create Release
+    runs-on: ubuntu-latest
+    outputs: 
+      upload_url: ${{ steps.create_release.outputs.upload_url }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # This token is provided by Actions
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: Release ${{ github.ref }}
+          draft: false
+          prerelease: false
+
+  build_wheels:
+    name: Wheels on ${{ matrix.os }}/py${{ matrix.python-version }}
+    needs: create_release
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [windows-latest, macos-latest]
+        python-version: [3.6, 3.7, 3.8, 3.9]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-python@v2
+        name: Install Python
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: What OS and Python version
+        run: |
+          uname -a
+          python --version
+          which python
+      - name: install pep517 and twine
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install pep517 twine
+      - name: Build wheels
+        run: |
+          python -m pep517.build --binary --out-dir dist/ .
+      - name: Display content dist folder
+        run: |
+          ls dist/
+      - name: Install and test distribution
+        env:
+          MPLBACKEND: agg
+        run: |
+          pip install --pre --find-links dist lumispy[tests]
+          pytest --pyargs lumispy
+      - uses: actions/upload-artifact@v2
+        with:
+          path: ./dist/*.whl
+
+      - name: Publish wheels to PyPI
+        if: github.repository_owner == 'lumispy'
+        env:
+          # Github secret set in the lumispy/lumispy repository
+          # Not available from fork or pull request
+          # Secrets are not passed to workflows that are triggered by a pull request from a fork
+          TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+          TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
```

Comment:
need to be added

Reply:
Sure. @jlaehne can you please create a user account in pypi and send me your user name?

Reply:
I have also added the TWINE_USERNAME and PASWORD as a secret for the repository.

Reply:
done